src/commands/uncommit.ts

```diff
@@ -42,9 +42,16 @@ export async function _uncommit(parsedSettings: ParsedSettings): Promise<void> {
   const contents = await fsp.readFile(lastMigrationFilepath, "utf8");
   const { headers, body } = parseMigrationText(lastMigrationFilepath, contents);
 
+  // Remove included migrations
+  const includeRegex =
+    /^--![ \t]*Included[ \t]+(?<filename>.*?\.sql)[ \t]*$.*?^--![ \t]*EndIncluded[ \t]*\k<filename>[ \t]*$/gms;
+  const decompiledBody = body.replace(includeRegex, (match) => {
+    return match.split("\n")[0].replace(" Included", "include");
+  });
```

Comment:
I'd rather use the capture group:

```suggestion
  const decompiledBody = body.replace(
    includeRegex,
    (_, filename) => `--! include ${filename}`,
  );
```